### PR TITLE
test: fix test freeze

### PR DESCRIPTION
**Pull request by alexander-akait**
- [ ] This is a **bugfix**
- [ ] This is a **feature**
- [ ] This is a **code refactor**
- [ ] This is a **test update**
- [ ] This is a **docs update**
- [x] This is a **metadata update**

### For Bugs and Features; did you add new tests?

it is tests

### Motivation / Use-Case

fix freeze and throw an good error if problem happens

### Breaking Changes

No

### Additional Info

fixes https://github.com/webpack/webpack-dev-server/pull/3399

### CODE REVIEW

File: test/server/liveReload-option.test.js [hunk 1]
```diff
@@ -1,124 +0,0 @@
-'use strict';
```
Comment (alexander-akait) [inline test/server/liveReload-option.test.js:1]: We don't need this stupid test anymore, we have good e2e tests right now https://github.com/webpack/webpack-dev-server/blob/master/test/e2e/hot-and-live-reload.test.js

File: test/server/webSocketServer-option.test.js [hunk 2]
```diff
@@ -389,7 +389,7 @@ describe('webSocketServer', () => {
         });
       });
 
-      describe('with a bad host header', () => {
+      describe.skip('with a bad host header', () => {
```
Comment (alexander-akait) [inline test/server/webSocketServer-option.test.js:392]: Ideally we should remove them in favor e2e test, I will add late them

**Comment by alexander-akait** [comment]
Should be fine now, in the next PR I will remove `ports-map.js` in favor simple function to get free port, so we will avoid problems with busy port and improve perf of out tests

**Review by anshumanv — DISMISSED** [review]

**Comment by alexander-akait** [comment]
Let's merge as is and remove `ports-map.js` in the next PR, it will fix the whole problem with busy ports and freezing test

**Pull request merged**